articles/active-directory/develop/developer-guide-conditional-access-authentication-context.md

```diff
@@ -216,6 +216,47 @@ Do not use auth context where the app itself is going to be a target of Conditio
 - [Use the Conditional Access auth context to perform step-up authentication for high-privilege operations in a web app](https://github.com/Azure-Samples/ms-identity-dotnetcore-ca-auth-context-app/blob/main/README.md)
 - [Use the Conditional Access auth context to perform step-up authentication for high-privilege operations in a web API](https://github.com/Azure-Samples/ms-identity-ca-auth-context/blob/main/README.md)
 
+## Authentication Context [ACRs] in Conditional Access expected behavior
+
+### Explicit Auth Context Satisfaction in Requests
+A client can explicitly ask for a token with an Auth Context (ACR) through the claims in the request's body. If an ACR was asked, Conditional Access will allow issuing the token with the asked ACR if all challenges were completed.
+
+### Expected Behavior When an Auth Context Is Not Protected by Conditional Access in The Tenant
+Conditional Access will allow an ACR addition to token's claims if there is no Conditional Access policy enforcing it.
```

Comment:
I'd add a bit of detail on why this is the case:


"Conditional Access will may issue an ACRS in a token's claims when all Conditional Access policy assigned to the ACRS value has been satisfied. If no Conditional Access policy is assigned to an ACRS value the claim may still be issued, because all policy requirements have been satisfied.

Reply:
Great comment. Maybe it should be under the "Explicit Auth Context Satisfaction in Requests" section?